### PR TITLE
Dw 1401 data migration test branch

### PR DESCRIPTION
**Pull request by mgilgar**

### CODE REVIEW

File: repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+  config:
+    /hippo:configuration/hippo:update/hippo:registry/DW-1284 - Migration from publication to series:
+      hipposys:batchsize: 10
+      hipposys:description: ''
+      hipposys:dryrun: false
+      hipposys:loglevel: TRACE
+      hipposys:parameters: "{\r\n    \"inputFile\" :  \"/content/assets/assetsmiguel/seriesinfoexport2.json/seriesinfoexport2.json/hippogallery:asset\"\
+        \ \r\n}"
+      hipposys:query: /jcr:root/content/documents/corporate-website//*[(@jcr:primaryType='publicationsystem:series')]
+      hipposys:script: "package org.hippoecm.frontend.plugins.cms.admin.updater\r\n\
```
Comment (GeoffHayward) [inline repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml:12]: This inline script should be moved to its own file.

**Review by GeoffHayward — COMMENTED** [review]
Just one comment.

File: repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml (hunk 1 above)
Comment (mgilgar) [reply repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml:12]: There is a DW-1284---Migration-from-publication-to-series.groovy file, is that what you mean?

Should I then remove the hipposys:script field and content in the yaml file?

Comment (GeoffHayward) [reply repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml]: Yes please.

Comment (GeoffHayward) [reply repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml]: You still need a YMAL file. You need one YMAL and one Groovy file per script.

Comment (mgilgar) [reply repository-data/application/src/main/resources/hcm-config/configuration/update/DW-1284---Migration-from-publication-to-series.yaml]: Ok, it is done. We still have to squash the changes, I will do it later.

**Comment by mgilgar** [comment]
This PR has been substituted by https://github.com/NHS-digital-website/hippo/pull/1210

**Pull request closed without merging**